pkg/client/attach.go

```diff
@@ -294,13 +302,35 @@ func (c *ConmonClient) handlePacket(cfg *AttachConfig, buf []byte, nr int) (cont
 	return true, nil
 }
 
+func (c *ConmonClient) tryCloseAttachReaderForID(id string) {
+	c.logger.Tracef("Closing attach reader for ID: %s", id)
+	if val, ok := c.attachReaders.Load(id); ok {
+		c.closeAttachReader(val)
+	}
+}
+
+func (c *ConmonClient) closeAttachReader(val any) {
```

Comment:
It seems like we can remove any and use a CloseReader, right?

Reply:
The intention was to catch the interface conversion as well because the sync map does not support generics yet.